### PR TITLE
feat(Hadoop) - Adding Apache Hadoop changelog

### PR DESCRIPTION
**Pull request by ChethanUK**
cc: @marc0der

### CODE REVIEW

**Comment by ChethanUK** [comment]
Can you review this @marc0der
Want to install both Hadoop 3.2 and 2.7 locally. Thanks :)

File: src/main/scala/io/sdkman/changelogs/HadoopMigrations.scala [hunk 1]
```diff
@@ -0,0 +1,70 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "059")
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/HadoopMigrations.scala:6]: This should be the next highest in order.
You can do this by running the following inside the project:
```
grep -R --no-filename 'ChangeLog(' | sort
```

**Review by marc0der — COMMENTED** [review]
Sorry for not merging sooner, I completely overlooked this PR @ChethanUK.

File: src/main/scala/io/sdkman/changelogs/HadoopMigrations.scala (hunk 1 above)
Comment (marc0der) [reply src/main/scala/io/sdkman/changelogs/HadoopMigrations.scala:6]: @ChethanUK lots of incoming traffic atm, can you make it `52` for me, please?

Comment (ChethanUK) [reply src/main/scala/io/sdkman/changelogs/HadoopMigrations.scala:6]: Hi Sorry, was bit busy. Didn't check this github acc much these days. I just realized I had deleted this repo's fork.
Created NEW PR: https://github.com/sdkman/sdkman-db-migrations/pull/524

**Review by marc0der — APPROVED** [review]

**Comment by marc0der** [comment]
Excellent, thanks for contributing!

**Comment by marc0der** [comment]
Looks like Hadoop is not distributed as a zip file and causes the installation to fail. I'll need to add a special case in the sdkman-hooks project...

**Comment by marc0der** [comment]
I've made the appropriate change in sdkman-hooks 1.1.20.

**Pull request merged**